tests/test_part_image.py

```diff
@@ -401,11 +401,11 @@ def test_expected_data_dirs(
                 # Expect /data/mender to exist always
                 assert "mender" in nodes
 
+                # Independently from mender-configure's installation status, we create the
+                # symlink for /var/lib/mender-configure to the data partition for mender-convert
+                # to support the subsequent installation of the add-on by the user
```

Comment:
why we add this comment here?

Reply:
It makes sense in this context, trust me :)